redash/models/__init__.py

```diff
@@ -579,12 +579,24 @@ def outdated_queries(cls):
     @classmethod
     def search(cls, term, group_ids, user_id=None, include_drafts=False,
                limit=None, include_archived=False):
+        from redash.authentication.org_resolving import current_org
         all_queries = cls.all_queries(
             group_ids,
             user_id=user_id,
             include_drafts=include_drafts,
             include_archived=include_archived,
         )
+
+        if current_org.get_setting("multi_byte_search_enabled"):
```

Comment:
I just noticed that all the search tests are failing, because now this code requires a request context to use `current_org`.

Let's make this an argument this method takes from the calling code (which has access to `current_org`).

Reply:
Fixed the tests by making `multi_byte_search` an argumeent.
Also added the model test for it.